docs/development/go/go-structures/index.md

```diff
@@ -18,235 +19,532 @@ external_resources:
 
 ## Introduction
 
-Although arrays, slices and maps are all very useful, they cannot group and hold values of multiple data types at the same place. When you need to group various types of variables and create new data types, you can use *structures* – the various elements of a structure are called *fields* of the structure or just fields. For reasons that will become clear in a while, the fields of a structure usually begin with an uppercase letter. Last, the order in which you put the fields in the definition of a structure type is significant for the type identity of the defined structure.
+Go's array, slice, and map types can be used to group multiple elements, but they cannot hold values of multiple data types. When you need to group different types of variables and create new data types, you can use *structs*.
 
 {{< note >}}
-This guide is written for a non-root user – none of the presented commands requires elevated privileges.
+Go does not have a concept of *classes* from other object oriented languages. Structs will be used in similar ways as classes, with important differences. For example, there is no class inheritance feature in Go.
```

Comment:
Added a short comparison/contrast to classes in other languages, for people familiar with classes already